Datetimeinterface (#1889)
sonata-project/SonataMediaBundle: UPGRADE-4.0.md

 * `GalleryHasMediaInterface::getId`
 * `GalleryInterface::getId`

> If you overrided some date-related methods (`setUpdatedAt`, `setCreatedAt`, `setCdnFlushAt`) in model classes (Gallery, Media, GalleryHasMedia),
> you need to change type hint of argument to `\DateTimeInterface`.

Reviewer: ```suggestion
If you have overridden some date-related methods (`setUpdatedAt()`, `setCreatedAt()`, `setCdnFlushAt()`) in model classes (`Gallery`, `Media`, `GalleryHasMedia`),
you need to change the arguments' type declarations to `\DateTimeInterface`.
```
